full-service/src/service/sync.rs

```diff
@@ -52,7 +56,12 @@ pub struct SyncThread {
 }
 
 impl SyncThread {
-    pub fn start(ledger_db: LedgerDB, wallet_db: WalletDb, logger: Logger) -> Self {
+    pub fn start(
+        ledger_db: LedgerDB,
+        wallet_db: WalletDb,
+        webhook_url: &str,
```

Comment:
⚠️  Signature change here - for strong typing, I would prefer to use `Url`, but at the same time, the reqwest library just takes a string